app/src/main/res/values/strings.xml

```diff
@@ -110,4 +110,13 @@
     <string name="stats_exp_for_a_week">points for a week</string>
     <string name="ex_dialog_continue">Continue</string>
     <string name="tutorial">Tutorial</string>
+    <string name="streak_success">%d in a row</string>
```

Comment:
change to
https://developer.android.com/guide/topics/resources/string-resource.html#Plurals